packages/shopify-cart/src/utils/formatCartResponse.ts

```diff
@@ -0,0 +1,39 @@
+import { CartResponse } from '../types/cart.type';
+import { Cart_CartFragment } from '../types/shopify.type';
+import { cartFromGql } from './';
+
+export type FormatCartResponseParams = Pick<
+  CartResponse,
+  'errors' | 'userErrors'
+> & {
+  cart: Cart_CartFragment | undefined | null;
+};
+
+/**
+ * Convert a `shopifyResponse` from a Storefront GraphQL response to a `CartResponse`
+ * @param params
+ * @returns Formatted `Cart` object
+ */
+export default function formatCartResponse({
+  cart,
+  userErrors,
+  errors
+}: FormatCartResponseParams): CartResponse {
+  const cartResponse: CartResponse = {
+    cart: null,
+    userErrors: null,
+    errors: null
+  };
+
+  cartResponse.errors = errors ?? null;
```

Comment:
Not a big deal at all, but might be worth considering using the same pattern for `errors` as is being used for `userErrors` & `cart`.

Reply:
All set